base/applications/rapps/geninst.cpp

```diff
@@ -152,6 +152,15 @@ ExtractCab(LPCWSTR Archive, const CStringW &OutputDir,
     return err ? err : ERROR_INTERNAL_ERROR;
 }
 
+
+static UINT
+ExtractArchive(LPCWSTR Archive, const CStringW &OutputDir, EXTRACTCALLBACK Callback, void *Cookie)
```

Comment:
"Cookie" -> "Context" ; this is a more standard naming for a user-specified parameter that gets passed to the Callback. (Cookie is mostly used for a "magic" number.)